src/test/resources/regressions/examples/disableNL.gobra

```diff
@@ -0,0 +1,11 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+package pkg
+
+requires 0 <= x && x <= 10
+requires 0 <= y && y <= 10
+ensures 0 <= res && res <= 100 // would fail with --disableNL
```

Comment:
Here, we can definitely add an annotation making this run with `--disableNL` and then we can add an expected post-condition failure